src/widgets/mentors-register/ui/mentors-register.module.scss

```diff
@@ -21,24 +25,9 @@
   margin-top: 40px;
 }
 
-.note {
-  margin-bottom: 18px;
-}
-
-.mentors-register-wrapper {
-  display: flex;
-  flex-direction: row;
-  gap: 20px;
-  justify-content: space-between;
-
-  @include media-tablet {
-    flex-direction: column;
-  }
-}
-
 .picture-wrapper {
   align-self: center;
```

Comment:
We don't need this. Also remove unnecessary wrapper here and add `width` & `height` to the `Image` class. Decrease sizes for `media-tablet`

Reply:
![Screenshot 2025-01-21 at 19 00 59](https://github.com/user-attachments/assets/e6c2ab0d-19e1-4d9e-bb87-efc8f20b79d8)